extensions/query/src/test/java/org/radargun/util/QueryTraitRepository.java

```diff
@@ -61,93 +61,93 @@ public Builder(ConcurrentHashMap cache) {
          }
 
          @Override
-         public Query.Builder subquery() {
+         public org.radargun.traits.Query.Builder subquery() {
```

Comment:
Why is this needed? Can't you import the class with the name `Query` to avoid this on line 3 above?

Reply:
It should be good now